onnxruntime/core/optimizer/selectors_actions/helpers.h

```diff
@@ -5,20 +5,49 @@
 
 namespace onnxruntime {
 
+// if the last input in num_inputs is for the variadic input, the variadic input could have zero or more values
```

Comment:
>num_inputs [](http://example.com/codeflow?start=24&length=10)

would this be generally useful? or should it be in helpers.cc?
nit: comment may need updating